webapp/src/components/post_type_jitsi/index.ts

```diff
@@ -0,0 +1,43 @@
+import {connect} from 'react-redux';
+import {bindActionCreators, Dispatch} from 'redux';
+
+import {getBool, getTheme} from 'mattermost-redux/selectors/entities/preferences';
+import {GlobalState} from 'mattermost-redux/types/store';
+import {GenericAction} from 'mattermost-redux/types/actions';
+
+import {Post} from 'mattermost-redux/types/posts';
+
+import {id as pluginId} from '../../manifest';
+import {displayUsernameForUser} from '../../utils/user_utils';
+import {enrichMeetingJwt, openJitsiMeeting} from '../../actions';
+
+import {PostTypeJitsi} from './post_type_jitsi';
+
+type OwnProps = {
+    post: Post,
+}
+
+function mapStateToProps(state: GlobalState, ownProps: OwnProps) {
+    const post = ownProps.post;
+    const user = state.entities.users.profiles[post.user_id];
+    const config = (state as any)[`plugins-${pluginId}`].config;
```

Comment:
Is `any`the only reasonable way to access this value?

Reply:
fixing it in another commit.